pkgs/top-level/all-packages.nix

```diff
@@ -23838,7 +23838,9 @@ with pkgs;
 
   gomodifytags = callPackage ../development/tools/gomodifytags { };
 
-  go-langserver = callPackage ../development/tools/go-langserver { };
+  go-langserver = callPackage ../development/tools/go-langserver {
```

Comment:
go-langserver is "deprioritised" in favour of gopls. Given that it will likely never work with go1.18 and generics, maybe it should just be removed in nixpkgs, see https://github.com/sourcegraph/go-langserver